mdal/frmts/mdal_selafin.hpp

```diff
@@ -48,6 +59,124 @@ namespace MDAL
       std::ifstream mIn;
   };
 
+  class SelafinDataset : public Dataset2D
```

Comment:
Would it be possible to add some developer documentation to the Selafin header, similar to what we have to 2dm and so. So developer can read how the data are written, if there is some official standard/doc with reference and so on?